### PR TITLE
Fix assets vercel uploading 

### PR DESCRIPTION
**Pull request by istarkov**
## Description

1. Add 404 instead of redirect
2. Fix vercel/remix upload

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: apps/builder/app/routes/$.tsx [hunk 1]
```diff
@@ -9,6 +13,17 @@ import { ErrorMessage } from "~/shared/error";
 import { getBuildParams, dashboardPath } from "~/shared/router-utils";
 
 export const loader = async ({ request }: LoaderArgs) => {
+  const url = new URL(request.url);
+  // See remix.config.ts for the publicPath value
+  const publicPath = "/build/";
```
Comment (kof) [inline apps/builder/app/routes/$.tsx:18]: shouldn't we get this value over some config or at least put it to some reusable place

**Review by TrySound — APPROVED** [review]

File: apps/builder/app/routes/$.tsx (hunk 1 above)
Comment (istarkov) [reply apps/builder/app/routes/$.tsx:18]: js and ts mix and moreover esm and cjs
between this file and remix.config.js

Looks like too many things to change is tsconfig, and usually causer runtime checks on exports as they are different in esm and cjs

Comment (istarkov) [reply apps/builder/app/routes/$.tsx:18]: cc @TrySound do you know a simple way to share const between remix.config.js and routes/$.ts?

Comment (TrySound) [reply apps/builder/app/routes/$.tsx:18]: I'm not sure. Some shit will happen for sure. Better hardcode. I'm sure this is temporary.

**Pull request merged**